lib/errors/MaxRetryExceededError.js

```diff
@@ -5,8 +5,9 @@ var NAME = "MaxRetryExceededError";
  *
  * @private
  */
-function MaxRetryExceededError() {
+function MaxRetryExceededError(missingPaths) {
     this.message = "The allowed number of retries have been exceeded.";
+    this.missingPaths = missingPaths || [];
     this.stack = (new Error()).stack;
```

Comment:
Since we're in the client this may not be possible, but is there something like `captureStackTrace` available across different browsers?

Reply:
To remain cross platform I think `(new Error()).stack` is the best that can be done here.